src/authn/ccloudProvider.test.ts

```diff
@@ -9,16 +9,15 @@ import {
 } from "../../tests/unit/testResources/connection";
 import { getTestExtensionContext, getTestStorageManager } from "../../tests/unit/testUtils";
 import { Connection } from "../clients/sidecar";
-import { CCLOUD_CONNECTION_ID } from "../constants";
+import { CCLOUD_AUTH_CALLBACK_URI, CCLOUD_CONNECTION_ID } from "../constants";
 import { getSidecar } from "../sidecar";
-import * as connections from "../sidecar/connections";
+import * as ccloud from "../sidecar/connections/ccloud";
+import * as watcher from "../sidecar/connections/watcher";
 import { getStorageManager, StorageManager } from "../storage";
 import { SecretStorageKeys } from "../storage/constants";
 import { getUriHandler, UriEventHandler } from "../uriHandler";
 import { ConfluentCloudAuthProvider, getAuthProvider } from "./ccloudProvider";
 
-const AUTH_CALLBACK_URI = vscode.Uri.parse("vscode://confluentinc.vscode-confluent/authCallback");
```

Comment:
Unrelated cleanup since https://github.com/confluentinc/vscode/pull/915 added the constant